src/js/components/Notification.js

```diff
@@ -76,11 +76,12 @@ export default class Notification extends Component {
     }
 
     let boxProps = Props.pick(this.props, Object.keys(Box.propTypes));
+    let fullBox = boxProps.hasOwnProperty('full') ? boxProps.full : 'horizontal';
```

Comment:
Because `full={false}` would always set the Box to 'horizontal'. We want `false` to be passed on to Box and 'horizontal' to only be set if the full prop is absent.